core-services/prow/03_deployment/deck.yaml

```diff
@@ -93,6 +93,7 @@ objects:
           - --job-config-path=/etc/job-config
           - --spyglass=true
           - --rerun-creates-job=true
+          - --gcs-credentials-file=/etc/gce-sa-credentials-gcs-publisher/service-account.json
```

Comment:
GCS publisher account should _not_ be the one that has access to the private bucket. `gcs-publisher` is for publishing (not reading) for `origin-ci-test`.